backend/attendance/CHANGELOG.md

```
@@ -1,6 +1,22 @@
 
 All notable changes to this module will be documented in this file.
 
+
+## 1.1.1 - 2025-06-13
+### Changes
+
+- Added new tag field in eg_wms_attendance_attendee table with supporting index 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)